migrations/migrationHelpers.js

```diff
@@ -117,11 +122,40 @@ const getdeployedContractInstance = async (
   return { Contract, contractInstance };
 };
 
+const source = 'https://raw.githubusercontent.com/razor-network/datasources/master';
+
+const getJobs = async () => {
+  try {
+    const jobs = await axios.get(`${source}/jobs.json`);
+    return jobs.data;
+  } catch (error) {
+    console.log(error.response.body);
```

Comment:
Be more descriptive here. `console.log("Error while fetching jobs: ", error.response.body);`

Reply:
resolved 57925d7